cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js

```diff
@@ -222,4 +228,35 @@ export default {
     }
     cy.do(saveButton.click());
   },
+
+  fillMappingProfileForUpdatesMarc:(specialMappingProfile = defaultMappingProfile) => {
+    cy.do([
+      TextField({ name:'profile.name' }).fillIn(specialMappingProfile.name),
+      Select({ name:'profile.incomingRecordType' }).choose(incomingRecordType.marcBib),
+      Select({ name:'profile.existingRecordType' }).choose(specialMappingProfile.typeValue),
+      Select({ name:'profile.mappingDetails.marcMappingOption' }).choose(fieldMappingsForMarc.update),
+    ]);
+  },
+
+  fillMappingProfileForInstance:(specialMappingProfile = defaultMappingProfile) => {
+    cy.do([
+      TextField({ name:'profile.name' }).fillIn(specialMappingProfile.name),
+      Select({ name:'profile.incomingRecordType' }).choose(incomingRecordType.marcBib),
+      Select({ name:'profile.existingRecordType' }).choose(specialMappingProfile.typeValue),
```

Comment:
MOve select  to const

Reply:
Please move TextField({ name:'profile.name' }) to const too

Reply:
these are different selects